CONTRIBUTING.md

```diff
@@ -0,0 +1,379 @@
+As a reminder, we expect all contributions to `aroma` to adhere to our [code of conduct][link_coc].
+
+## Contributing small documentation changes
+If you are new to GitHub and just have a small documentation change
+recommendation, please submit it to [our e-mail address](mailto:aroma.devs@gmail.com)
+and one of our developers will add it to the documentation directly.
+
+## Contributing through GitHub
+
+[git][link_git] is a really useful tool for version control.
+[GitHub][link_github] sits on top of git and supports collaborative and distributed working.
+
+You'll use [Markdown][markdown] to chat in issues and pull requests on GitHub.
+You can think of Markdown as a few little symbols around your text that will allow GitHub
+to render the text with a little bit of formatting.
+For example you could write words as bold (`**bold**`), or in italics (`*italics*`),
+or as a [link][rick_roll] (`[link](https://https://youtu.be/dQw4w9WgXcQ)`) to another webpage.
+
+GitHub has a helpful page on
+[getting started with writing and formatting Markdown on GitHub][writing_formatting_github].
+
+
+## Understanding issues, milestones and project boards
+
+Every project on GitHub uses [issues][link_issues], [milestones][link_milestones],
+and [project boards][link_project_boards] slightly differently.
+
+The following outlines how the ``aroma`` developers think about these different tools.
+
+* **Issues** are individual pieces of work that need to be completed to move the project forwards.
+A general guideline: if you find yourself tempted to write a great big issue that
+is difficult to describe as one unit of work, please consider splitting it into two or more issues.
+
+    Issues are assigned [labels](#issue-labels) which explain how they relate to the overall project's
+    goals and immediate next steps.
+
+    Sometimes issues may not produce action items, and conversation will stall after a few months.
+    When this happens, they may be marked stale by [stale-bot][link_stale-bot],
+    and will be closed after a week unless there is more discussion.
+    This helps us keep the issue tracker organized.
+    Any new discussion on the issue will remove the `stale` label, and prevent it from closing.
+    So, if theres's a discussion you think it not yet resolved, please jump in !
+
+* **Milestones** are the link between the issues and the high level strategy for the ``aroma`` project.
+Contributors new and old are encouraged to take a look at the milestones to see how we are progressing
+towards ``aroma``'s shared vision.
+
+    Issues are assigned to these milestones by the maintainers.
+    If you feel that an issue should be assigned to a specific milestone but the maintainers have not done so, just ask!
+    We might have just missed it, or we might not (yet) see how it aligns with the overall project structure.
+    These conversations are important to have, and we are excited to hear your perspective!
```

Comment:
I'd remove the idea of milestones. This repository is not an ongoing project in the same way `tedana` is (i.e., there are specific goals we want to accomplish and that's basically it), so we could just list those goals directly rather than talking about milestones in the abstract.